nailgun/entities.py

```diff
@@ -1042,6 +1042,7 @@ def __init__(self, server_config=None, **kwargs):
             'attach_stdin': entity_fields.BooleanField(),
             'attach_stdout': entity_fields.BooleanField(),
             'command': entity_fields.StringField(
+                default='top',
```

Comment:
When you say this you are looking just for Robottelo. Since this is something on Robottelo I think we can't assume that on NailGun for all users.

Also depending on the image top may not be available and if the user forget to fill the command it will use top automatically and will not enforce the user on providing a command.

I am ok on setting this default because almost images have the top, I remember having issues with some images though. I just left some thoughts related to the NailGun users, if others agree on merging with the default value I am ok.